test/common/network/address_impl_test.cc

```diff
@@ -392,6 +392,62 @@ TEST(AddressFromSockAddr, Pipe) {
 #endif
 }
 
+// Test comparisons between all the different (known) test classes.
+TEST(MixedAddresses, Equality) {
+  enum InstanceType { kIpv4, kIpv6, kPipe };
```

Comment:
nit: we don't use 'k' prefix for constance. I would just drop the 'k'.

Reply:
Done.